src/style.css

```
@@ -0,0 +1,105 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+  color: #3d4e61;
+}
+
+.input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+  font-size: 16px;
+}
+
+.inputSmall::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+  font-size: 14px;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  box-sizing: border-box;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline-style: none;
+
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 24px;
+
+  font-family: Avenir, sans-serif;
```

set it to `html` tag selector and let input inherit it 
```suggestion
  font-family: inherit;
```